firefox-ios/Client/Application/AppLaunchUtil.swift

```diff
@@ -33,9 +35,12 @@ class AppLaunchUtil {
         TelemetryWrapper.shared.setup(profile: profile)
         recordStartUpTelemetry()
 
-        // Need to get "settings.sendUsageData" this way so that Sentry can be initialized before getting the Profile.
-        let sendUsageData = NSUserDefaultsPrefs(prefix: "profile").boolForKey(AppConstants.prefSendUsageData) ?? true
-        logger.setup(sendUsageData: sendUsageData)
+        // Need to get "settings.sendCrashReports" this way so that Sentry can be initialized before getting the Profile.
+        let sendCrashReports = NSUserDefaultsPrefs(prefix: "profile").boolForKey(AppConstants.prefSendCrashReports) ?? true
```

Comment:
Shouldn't this be default false?

Reply:
This is a product request. Option should be true by default. :)